pom.xml

```diff
@@ -1136,7 +1136,7 @@
         <exclusions>
           <exclusion>
             <groupId>org.scala-lang.modules</groupId>
-            <artifactId>scala-xml_2.13</artifactId>
+            <artifactId>scala-xml_${scala.binary.version}</artifactId>
```

Comment:
If this is not required, shall we remove this in order to simplify the patch?

IIUC, this is correct, but we already have many hard-coded ones in Apache Spark `pom.xml` files.
```
$ git grep '_2.13' | grep pom.xml | wc -l
      88
```

So, if we need this, we can proceed in a separate independent PR.

Reply:
we should exclude `<artifactId>spark-*</artifactId>`, then it's the only one.
```
$ git grep '_2.13' | grep pom.xml | grep -v '<artifactId>spark-' | wc -l
1
```
Change is not necessary, I can revert it.

Reply:
Then, please make a new PR for that change, @pan3793 . It's also meaningful that you verified this is the last one. :)